templates/Template k8s-patroni.xml

```diff
@@ -280,13 +280,13 @@ return(timestampSeconds)</parameter>
                                     <expression>change(/Template k8s-patroni postgres status/k8s-patroni.state[{#CONTEXT},{#NAMESPACE},{#CONFIGMAP},{#MEMBER}])=1</expression>
                                     <name>Patroni postgres {#MEMBER} state changed in namespace {#NAMESPACE}</name>
                                     <priority>HIGH</priority>
-                                    <description>trigger for patroni postgres state change</description>
+                                    <description>Trigger for patroni postgres state change</description>
```

Comment:
Hehe. These descriptions are kind of useless. We never bothered to fill those out because the \<name> tells the story. We only populate them when there is something "different" to be aware of.

But, no need to remove them now.